### PR TITLE
tikv-client: reducing message of resolved ts

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <tangminghua@pingcap.com>

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update the `tidb-cdc/cdc-ansible`

### Release note

* tikv-client: reducing message of resolved ts

### CODE REVIEW

**Comment by amyangfei** [comment]
Please resolve conflicts

File: cdc/kv/client.go [hunk 1]
```diff
@@ -1192,147 +1283,132 @@ func (s *eventFeedSession) singleEventFeed(
...
-			checkpointTs = x.ResolvedTs
+		}
 
-			select {
-			case s.eventCh <- revent:
-				metricSendEventResolvedCounter.Inc()
-			case <-ctx.Done():
-				return checkpointTs, errors.Trace(ctx.Err())
+		if event.resolvedTs != nil {
+			if err := handleResolvedTs(event.resolvedTs.Ts); err != nil {
```
Comment (amyangfei) [inline cdc/kv/client.go:1410]: The `ResolvedTs` event contains a list of region ids, but it is not used now.
Will the `ResolvedTs` event be sent to each region in this region list, then the count of resolved ts event seems not reduced.

Comment (5kbpers) [reply cdc/kv/client.go:1410]: Sure, it's for reducing the message from tikv to ticdc.
If we want to reduce the event inside ticdc, the tikv client may need to be carefully refactored.

**Comment by amyangfei** [comment]
Please fix CI

**Comment by 5kbpers** [comment]
close it because of too many conflicts, have manually picked and opened a new one.

**Pull request closed without merging**